packages/commercetools/api-client/src/api/getProduct/index.ts

```diff
@@ -1,7 +1,5 @@
-import { ApolloQueryResult } from 'apollo-client';
 import gql from 'graphql-tag';
-import { apolloClient, currency, country, locale, acceptLanguage } from './../../index';
-import { ProductSearch } from './../../types/Api';
+import { acceptLanguage, apolloClient, country, currency, locale } from './../../index';
```

Comment:
will conflict with https://github.com/DivanteLtd/vue-storefront/pull/4791 - please wait until its merged and correct accordingly

Reply:
ok, will wait for it